env.list.sample

```diff
@@ -1,3 +1,4 @@
 DISCORD_TOKEN=<YOUR DISCORD TOKEN>
 DISCORD_BOT_ID=<bot id>
+DISCORD_BOT_PUBLIC_KEY=<BOT public key>
```

Comment:
Bot's public key for this signature verification.
https://discord.com/developers/docs/interactions/receiving-and-responding#security-and-authorization